Add proxies parameters when downloading the model

TRHX opened the pull request:
Add proxy parameters when downloading the model to make it more convenient for Chinese users to use, the proxy parameter is optional. #638 

```python
>>> import stanza
>>> proxies = {'http': 'http://127.0.0.1:10809', 'https': 'http://127.0.0.1:10809'}
>>> stanza.download('en', proxies)
```

AngledLuffa commented:
Would you make this the last argument of the parameters list instead of the first?  Also, the .idea directory isn't supposed to be here, I believe.

TRHX commented:
> Would you make this the last argument of the parameters list instead of the first? Also, the .idea directory isn't supposed to be here, I believe.

I have modified the location of the proxy parameter to the last one, and deleted the .idea directory.

AngledLuffa commented:
Thank you!
